src/agent/Core/ApplicationPool/Group/InternalUtils.cpp

```diff
@@ -316,6 +316,18 @@ Group::testOverflowRequestQueue() const {
 	}
 }
 
+bool
+Group::testTimeoutRequestQueue() const {
+	// This has a performance penalty, although I'm not sure whether the penalty is
```

Comment:
I don't think this has a performance penalty. Looking up debugSupport information is pretty cheap.